test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -1274,5 +1274,175 @@ public static void CreateTempGlobalJson(string solutionRoot)
+        <FileAlignment>512</FileAlignment>
+        <RootNamespace>$NAME$</RootNamespace>
+        <AssemblyName>$NAME$</AssemblyName>
+        <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
+        </PropertyGroup>
+        <Import Project=""$(NanoFrameworkProjectSystemPath)NFProjectSystem.props"" Condition=""Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.props')"" />
+        <ItemGroup>");
+
+            foreach ((string, string) package in packages)
+            {
+                projContent.AppendLine(
+$@"
+        <Reference Include=""{package.Item1}, Version={package.Item2}, Culture=neutral"">
+            <HintPath>..\packages\{package.Item1}.{package.Item2}\lib\{package.Item1}.dll</HintPath>
+            <Private>True</Private>
+            <SpecificVersion>True</SpecificVersion>
+        </Reference>");
+            }
+
+            projContent.AppendLine(
+@"
+        </ItemGroup>
+            <ItemGroup>
+            <None Include=""packages.config"" />
+            </ItemGroup>
+            <Import Project=""$(NanoFrameworkProjectSystemPath)NFProjectSystem.CSharp.targets"" Condition=""Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.CSharp.targets')"" />
+            <ProjectExtensions>
+            <ProjectCapabilities>
+                <ProjectConfigurationsDeclaredAsItems />
+            </ProjectCapabilities>
+            </ProjectExtensions>
+        </Project>".Replace("$NAME$", projectName));
+
+            return projContent.ToString();
+        }
+
+        /// <summary>
+        /// Create a simple package with a lib folder for .NET nanoFramework
+        /// The package will be removed from the machine cache upon creation
+        /// </summary>
+        public static ZipPackage CreateNFPackage(string repositoryPath, string id, string version)
+        {
+            PackageBuilder package = Util.CreateTestPackageBuilder(id, version);
+            IPackageFile libFile = Util.CreatePackageFile("lib/a.dll", "a");
+            package.Files.Add(libFile);
+
+            string path = Util.CreateTestPackage(package, repositoryPath);
+
+            var zipPackage = new ZipPackage(path);
+
+            return zipPackage;
+        }
+
+        /// <summary>
+        /// Create a Solution file with .NET nanoFramework projects in it.
+        /// </summary>
+        /// <param name="projectList">List of .NET nanoFramework projects to add to the solution</param>
+        /// <returns>Content of the Solution file</returns>
+        public static string CreateNFSolutionFileContent(List<string> projectList)
+        {
+            var slnContent = new StringBuilder();
+
+            slnContent.AppendLine("Microsoft Visual Studio Solution File, Format Version 12.00");
+            slnContent.AppendLine("# Visual Studio Version 16");
+            slnContent.AppendLine("VisualStudioVersion = 16.0.31005.135");
+
+            foreach(string project in projectList)
```

Comment:
space after foreach

Reply:
no more space

Reply:
Please check below example, we want space `foreach (` : 

https://github.com/NuGet/NuGet.Client/blob/73e59773c438657a1a9bf1fb3d8ef0f3851566e1/src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs#L969-L974

Another way to deal with syntax is install  `dotnet-format` tool from https://github.com/dotnet/format
then run `dotnet-format --exclude submodules`, it'll take care of it for you.

Reply:
Got you. Should be OK now